packages/core/src/App/AppContent.tsx

```diff
@@ -38,6 +38,20 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
     const { tracking_datadog } = data;
     const is_passkeys_supported = browserSupportsWebAuthn();
 
+    const livechat_client_information: Parameters<typeof useLiveChat>[0] = {
+        is_client_initialized: store.client.is_client_initialized,
```

Comment:
Absolutely optional, but above you extracted some values from `store.client` and `store.common`, so we can do the same here, in order to get rid of duplications `store.client`. Please, ignore if irrelevant